packages/api/endpoints/pdrs.js

```diff
@@ -63,12 +67,16 @@ async function del(req, res) {
 
   const pdrS3Key = `${process.env.stackName}/pdrs/${pdrName}`;
 
-  await deleteS3Object(process.env.system_bucket, pdrS3Key);
-
   const pdrModel = new models.Pdr();
+  const pdrPgModel = new PdrPgModel();
+  const knex = await getKnexClient();
 
   try {
-    await pdrModel.delete({ pdrName });
+    await knex.transaction(async (trx) => {
+      await pdrPgModel.delete(trx, { name: pdrName });
+      await pdrModel.delete({ pdrName });
+      await deleteS3Object(process.env.system_bucket, pdrS3Key);
```

Comment:
So previously the S3 file was deleted before the dynamo object and now that order is reversed, which means you could have a record deleted from Dynamo but still have an S3 object if the S3 delete failed. That's fine with me as long as it is intentional

Reply:
I wasn't sure about the ordering on that. I do think now it would be preferable to reverse it. I'll push a change here in a minute.